app/src/main/java/ch/hikemate/app/utils/MapUtils.kt

```diff
@@ -75,9 +86,11 @@ object MapUtils {
             true
           })
         }
-
-    mapView.overlays.add(line)
+    // The index provides the lowest priority so that the facilities and other overlays
+    // are always displayed on top of it.
+    mapView.overlays.add(ROUTE_PRIORITY_DISPLAY, line)
     mapView.overlays.add(startingMarker)
+    mapView.invalidate()
```

Comment:
Question: What does that do? Do we need it ?

Reply:
This forces the map to be redrawn so that the overlays are always **on top** of the Hike Route. This fixed the bug of the facilities being drawn behind the Route. I will update the name of the PR to account for this.